lib/client/client-spec.js

```diff
@@ -0,0 +1,79 @@
+var mediator = require('fh-wfm-mediator/lib/mediator');
+var sinon = require('sinon');
+var assert = require('assert');
+
+describe("Raincatcher Sync Client Initialisation", function() {
+
+  var mockSyncOptions = {
+    "sync_frequency": 5,
+    "storage_strategy": "dom",
+    "do_console_log": false
+  };
+
+  var mockDataSetId = "mockdatasetid";
+
+  var mockDataSetFilterParams = {filterParam1: "filtervalue"};
+  var mockDataSetMetadataParams = {metadataParam1: "metadatavalue"};
+
+  before(function() {
```

Comment:
Wouldn't that mean that the order of your tests matter now? I can't think of any time when sharing the same stub instances among tests is ever a good idea. Each test should be completely independent of all others.